definitions.yml

```
@@ -537,6 +537,35 @@ definitions:
         description: "Map the value setted in Zendesk"
       description:
         $ref: "#/definitions/ZendeskDescriptionMessage"
+  VersionInfo:
+    type: object
+    description: "Information related to the app version"
+    required:
+      - min_app_version
+      - latest_released_app_version
+      - rollout_app_version
+    properties:
+      min_app_version:
+        $ref: "#/definitions/VersionPerPlatform"
+      min_app_version_pagopa:
+        $ref: "#/definitions/VersionPerPlatform"
+      latest_released_app_version:
+        $ref: "#/definitions/VersionPerPlatform"
+      rollout_app_version:
+        $ref: "#/definitions/VersionPerPlatform"
+  VersionPerPlatform:
+    type: object
+    description: "Associate an app version with a platform "
+    required:
+      - ios
+      - android
+    properties:
+      ios:
+        type: string
```

```suggestion
        type: string
        pattern: '^\\d+.\\d+.\\d+(.\\d+)?$'
```
What do you think to keep this string close to version format?
In this way we'll have this codec generated:

```typescript
const VersionPerPlatformR = t.interface({
  ios: PatternString("^\\d+.\\d+.\\d+(.\\d+)?$"),

  android: PatternString("^\\d+.\\d+.\\d+(.\\d+)?$")
});
```

And if someone edit the json like this:
```
"rollout_app_version" : {
    "ios": "0.0.0",
    "android": "ciao"
  }
```
the check will fail

PS: I tested the proposed change

---

The stricter the better! Didn't know this further check, added in 5cb5a4be4cd7b67b0804b9d3a5a02e062b2d4cda